2017/07/11 10:47:50 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "login".' in C:\xampp\htdocs\xinjian\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\xinjian\framework\web\CWebApplication.php(141): CWebApplication->runController('login')
#1 C:\xampp\htdocs\xinjian\framework\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\xampp\htdocs\xinjian\xinde\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/xinjian/xinde/index.php?r=login
---
2017/07/11 10:52:52 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "login".' in C:\xampp\htdocs\xinjian\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\xinjian\framework\web\CController.php(270): CController->missingAction('login')
#1 C:\xampp\htdocs\xinjian\framework\web\CWebApplication.php(282): CController->run('login')
#2 C:\xampp\htdocs\xinjian\framework\web\CWebApplication.php(141): CWebApplication->runController('User/login')
#3 C:\xampp\htdocs\xinjian\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\xinjian\xinde\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/xinjian/xinde/index.php?r=User/login
---
2017/07/11 10:53:18 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "login".' in C:\xampp\htdocs\xinjian\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\xinjian\framework\web\CController.php(270): CController->missingAction('login')
#1 C:\xampp\htdocs\xinjian\framework\web\CWebApplication.php(282): CController->run('login')
#2 C:\xampp\htdocs\xinjian\framework\web\CWebApplication.php(141): CWebApplication->runController('user/login')
#3 C:\xampp\htdocs\xinjian\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\xinjian\xinde\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/xinjian/xinde/index.php?r=user/login
---
2017/07/11 10:54:39 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "login".' in C:\xampp\htdocs\xinjian\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\xinjian\framework\web\CController.php(270): CController->missingAction('login')
#1 C:\xampp\htdocs\xinjian\framework\web\CWebApplication.php(282): CController->run('login')
#2 C:\xampp\htdocs\xinjian\framework\web\CWebApplication.php(141): CWebApplication->runController('user/login')
#3 C:\xampp\htdocs\xinjian\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\xinjian\xinde\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/xinjian/xinde/index.php?r=user/login
---
2017/07/11 10:54:41 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "login".' in C:\xampp\htdocs\xinjian\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\xinjian\framework\web\CController.php(270): CController->missingAction('login')
#1 C:\xampp\htdocs\xinjian\framework\web\CWebApplication.php(282): CController->run('login')
#2 C:\xampp\htdocs\xinjian\framework\web\CWebApplication.php(141): CWebApplication->runController('user/login')
#3 C:\xampp\htdocs\xinjian\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\xinjian\xinde\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/xinjian/xinde/index.php?r=user/login
---
2017/07/11 10:57:02 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "login".' in C:\xampp\htdocs\xinjian\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\xinjian\framework\web\CController.php(270): CController->missingAction('login')
#1 C:\xampp\htdocs\xinjian\framework\web\CWebApplication.php(282): CController->run('login')
#2 C:\xampp\htdocs\xinjian\framework\web\CWebApplication.php(141): CWebApplication->runController('user/login')
#3 C:\xampp\htdocs\xinjian\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\xinjian\xinde\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/xinjian/xinde/index.php?r=user/login
---
2017/07/11 10:57:04 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "login".' in C:\xampp\htdocs\xinjian\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\xinjian\framework\web\CController.php(270): CController->missingAction('login')
#1 C:\xampp\htdocs\xinjian\framework\web\CWebApplication.php(282): CController->run('login')
#2 C:\xampp\htdocs\xinjian\framework\web\CWebApplication.php(141): CWebApplication->runController('user/login')
#3 C:\xampp\htdocs\xinjian\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\xinjian\xinde\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/xinjian/xinde/index.php?r=user/login
---
2017/07/11 10:57:17 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "login".' in C:\xampp\htdocs\xinjian\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\xinjian\framework\web\CController.php(270): CController->missingAction('login')
#1 C:\xampp\htdocs\xinjian\framework\web\CWebApplication.php(282): CController->run('login')
#2 C:\xampp\htdocs\xinjian\framework\web\CWebApplication.php(141): CWebApplication->runController('user/login')
#3 C:\xampp\htdocs\xinjian\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\xinjian\xinde\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/xinjian/xinde/index.php?r=user/login
---
2017/07/11 10:57:19 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "login".' in C:\xampp\htdocs\xinjian\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\xinjian\framework\web\CController.php(270): CController->missingAction('login')
#1 C:\xampp\htdocs\xinjian\framework\web\CWebApplication.php(282): CController->run('login')
#2 C:\xampp\htdocs\xinjian\framework\web\CWebApplication.php(141): CWebApplication->runController('user/login')
#3 C:\xampp\htdocs\xinjian\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\xinjian\xinde\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/xinjian/xinde/index.php?r=user/login
---
2017/07/12 10:13:07 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [1049] Unknown database 'newtest'' in C:\xampp\htdocs\xinjian\framework\db\CDbConnection.php:381
Stack trace:
#0 C:\xampp\htdocs\xinjian\framework\db\CDbConnection.php(330): CDbConnection->open()
#1 C:\xampp\htdocs\xinjian\framework\db\CDbConnection.php(308): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\xinjian\framework\base\CModule.php(387): CDbConnection->init()
#3 C:\xampp\htdocs\xinjian\framework\base\CModule.php(103): CModule->getComponent('db')
#4 C:\xampp\htdocs\xinjian\xinde\protected\views\user\login.php(158): CModule->__get('db')
#5 C:\xampp\htdocs\xinjian\framework\web\CBaseController.php(126): require('C:\\xampp\\htdocs...')
#6 C:\xampp\htdocs\xinjian\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\\xampp\\htdocs...', NULL, true)
#7 C:\xampp\htdocs\xinjian\framework\web\CController.php(869): CBaseController->renderFile('C:\\xampp\\htdocs...', NULL, true)
#8 C:\xampp\htdocs\xinjian\xinde\protected\controllers\UserController.php(12): CController->renderPartial('login')
#9 C:\xampp\htdocs\xinjian\framework\web\actions\CInlineAction.php(49): UserController->actionLogin()
#10 C:\xampp\htdocs\xinjian\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#11 C:\xampp\htdocs\xinjian\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#12 C:\xampp\htdocs\xinjian\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#13 C:\xampp\htdocs\xinjian\framework\web\CWebApplication.php(282): CController->run('login')
#14 C:\xampp\htdocs\xinjian\framework\web\CWebApplication.php(141): CWebApplication->runController('user/login')
#15 C:\xampp\htdocs\xinjian\framework\base\CApplication.php(169): CWebApplication->processRequest()
#16 C:\xampp\htdocs\xinjian\xinde\index.php(13): CApplication->run()
#17 {main}
REQUEST_URI=/xinjian/xinde/index.php?r=user/login
---
2017/07/12 10:17:45 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [1049] Unknown database 'newtest'' in C:\xampp\htdocs\xinjian\framework\db\CDbConnection.php:381
Stack trace:
#0 C:\xampp\htdocs\xinjian\framework\db\CDbConnection.php(330): CDbConnection->open()
#1 C:\xampp\htdocs\xinjian\framework\db\CDbConnection.php(308): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\xinjian\framework\base\CModule.php(387): CDbConnection->init()
#3 C:\xampp\htdocs\xinjian\framework\base\CModule.php(103): CModule->getComponent('db')
#4 C:\xampp\htdocs\xinjian\xinde\protected\views\user\login.php(158): CModule->__get('db')
#5 C:\xampp\htdocs\xinjian\framework\web\CBaseController.php(126): require('C:\\xampp\\htdocs...')
#6 C:\xampp\htdocs\xinjian\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\\xampp\\htdocs...', NULL, true)
#7 C:\xampp\htdocs\xinjian\framework\web\CController.php(869): CBaseController->renderFile('C:\\xampp\\htdocs...', NULL, true)
#8 C:\xampp\htdocs\xinjian\xinde\protected\controllers\UserController.php(12): CController->renderPartial('login')
#9 C:\xampp\htdocs\xinjian\framework\web\actions\CInlineAction.php(49): UserController->actionLogin()
#10 C:\xampp\htdocs\xinjian\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#11 C:\xampp\htdocs\xinjian\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#12 C:\xampp\htdocs\xinjian\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#13 C:\xampp\htdocs\xinjian\framework\web\CWebApplication.php(282): CController->run('login')
#14 C:\xampp\htdocs\xinjian\framework\web\CWebApplication.php(141): CWebApplication->runController('user/login')
#15 C:\xampp\htdocs\xinjian\framework\base\CApplication.php(169): CWebApplication->processRequest()
#16 C:\xampp\htdocs\xinjian\xinde\index.php(13): CApplication->run()
#17 {main}
REQUEST_URI=/xinjian/xinde/index.php?r=user/login
---
2017/07/12 10:17:49 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [1049] Unknown database 'newtest'' in C:\xampp\htdocs\xinjian\framework\db\CDbConnection.php:381
Stack trace:
#0 C:\xampp\htdocs\xinjian\framework\db\CDbConnection.php(330): CDbConnection->open()
#1 C:\xampp\htdocs\xinjian\framework\db\CDbConnection.php(308): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\xinjian\framework\base\CModule.php(387): CDbConnection->init()
#3 C:\xampp\htdocs\xinjian\framework\base\CModule.php(103): CModule->getComponent('db')
#4 C:\xampp\htdocs\xinjian\xinde\protected\views\user\login.php(158): CModule->__get('db')
#5 C:\xampp\htdocs\xinjian\framework\web\CBaseController.php(126): require('C:\\xampp\\htdocs...')
#6 C:\xampp\htdocs\xinjian\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\\xampp\\htdocs...', NULL, true)
#7 C:\xampp\htdocs\xinjian\framework\web\CController.php(869): CBaseController->renderFile('C:\\xampp\\htdocs...', NULL, true)
#8 C:\xampp\htdocs\xinjian\xinde\protected\controllers\UserController.php(12): CController->renderPartial('login')
#9 C:\xampp\htdocs\xinjian\framework\web\actions\CInlineAction.php(49): UserController->actionLogin()
#10 C:\xampp\htdocs\xinjian\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#11 C:\xampp\htdocs\xinjian\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#12 C:\xampp\htdocs\xinjian\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#13 C:\xampp\htdocs\xinjian\framework\web\CWebApplication.php(282): CController->run('login')
#14 C:\xampp\htdocs\xinjian\framework\web\CWebApplication.php(141): CWebApplication->runController('user/login')
#15 C:\xampp\htdocs\xinjian\framework\base\CApplication.php(169): CWebApplication->processRequest()
#16 C:\xampp\htdocs\xinjian\xinde\index.php(13): CApplication->run()
#17 {main}
REQUEST_URI=/xinjian/xinde/index.php?r=user/login
---
2017/07/14 10:47:20 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "head.php".' in C:\xampp\htdocs\xinjian\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\xinjian\framework\web\CController.php(270): CController->missingAction('head.php')
#1 C:\xampp\htdocs\xinjian\framework\web\CWebApplication.php(282): CController->run('head.php')
#2 C:\xampp\htdocs\xinjian\framework\web\CWebApplication.php(141): CWebApplication->runController('houtai/index/he...')
#3 C:\xampp\htdocs\xinjian\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\xinjian\xinde\index.php(14): CApplication->run()
#5 {main}
REQUEST_URI=/xinjian/xinde/index.php?r=houtai/index/head.php
HTTP_REFERER=http://localhost/xinjian/xinde/index.php?r=houtai/index/index
---
2017/07/14 10:47:20 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "right.php".' in C:\xampp\htdocs\xinjian\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\xinjian\framework\web\CController.php(270): CController->missingAction('right.php')
#1 C:\xampp\htdocs\xinjian\framework\web\CWebApplication.php(282): CController->run('right.php')
#2 C:\xampp\htdocs\xinjian\framework\web\CWebApplication.php(141): CWebApplication->runController('houtai/index/ri...')
#3 C:\xampp\htdocs\xinjian\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\xinjian\xinde\index.php(14): CApplication->run()
#5 {main}
REQUEST_URI=/xinjian/xinde/index.php?r=houtai/index/right.php
HTTP_REFERER=http://localhost/xinjian/xinde/index.php?r=houtai/index/index
---
2017/07/14 10:47:20 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "left.php".' in C:\xampp\htdocs\xinjian\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\xinjian\framework\web\CController.php(270): CController->missingAction('left.php')
#1 C:\xampp\htdocs\xinjian\framework\web\CWebApplication.php(282): CController->run('left.php')
#2 C:\xampp\htdocs\xinjian\framework\web\CWebApplication.php(141): CWebApplication->runController('houtai/index/le...')
#3 C:\xampp\htdocs\xinjian\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\xinjian\xinde\index.php(14): CApplication->run()
#5 {main}
REQUEST_URI=/xinjian/xinde/index.php?r=houtai/index/left.php
HTTP_REFERER=http://localhost/xinjian/xinde/index.php?r=houtai/index/index
---
2017/07/20 10:15:29 [error] [exception.CException] exception 'CException' with message 'UserController and its behaviors do not have a method or closure named "renderPattial".' in C:\xampp\htdocs\xinjian\framework\base\CComponent.php:265
Stack trace:
#0 C:\xampp\htdocs\xinjian\xinde\protected\controllers\UserController.php(21): CComponent->__call('renderPattial', Array)
#1 C:\xampp\htdocs\xinjian\xinde\protected\controllers\UserController.php(21): UserController->renderPattial('ttest')
#2 C:\xampp\htdocs\xinjian\framework\web\actions\CInlineAction.php(49): UserController->actionTtest()
#3 C:\xampp\htdocs\xinjian\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 C:\xampp\htdocs\xinjian\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 C:\xampp\htdocs\xinjian\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 C:\xampp\htdocs\xinjian\framework\web\CWebApplication.php(282): CController->run('ttest')
#7 C:\xampp\htdocs\xinjian\framework\web\CWebApplication.php(141): CWebApplication->runController('user/ttest')
#8 C:\xampp\htdocs\xinjian\framework\base\CApplication.php(169): CWebApplication->processRequest()
#9 C:\xampp\htdocs\xinjian\xinde\index.php(14): CApplication->run()
#10 {main}
REQUEST_URI=/xinjian/xinde/index.php?r=user/ttest
---
2017/07/20 11:48:09 [error] [php] array_shift() expects parameter 1 to be array, string given (C:\xampp\htdocs\xinjian\xinde\protected\views\user\ttest.php:10)
Stack trace:
#0 C:\xampp\htdocs\xinjian\framework\web\CBaseController.php(95): UserController->renderInternal()
#1 C:\xampp\htdocs\xinjian\framework\web\CController.php(869): UserController->renderFile()
#2 C:\xampp\htdocs\xinjian\xinde\protected\controllers\UserController.php(24): UserController->renderPartial()
#3 C:\xampp\htdocs\xinjian\framework\web\actions\CInlineAction.php(49): UserController->actionTtest()
#4 C:\xampp\htdocs\xinjian\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\xampp\htdocs\xinjian\framework\web\CController.php(286): UserController->runAction()
#6 C:\xampp\htdocs\xinjian\framework\web\CController.php(265): UserController->runActionWithFilters()
#7 C:\xampp\htdocs\xinjian\framework\web\CWebApplication.php(282): UserController->run()
#8 C:\xampp\htdocs\xinjian\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 C:\xampp\htdocs\xinjian\framework\base\CApplication.php(169): CWebApplication->processRequest()
#10 C:\xampp\htdocs\xinjian\xinde\index.php(14): CWebApplication->run()
REQUEST_URI=/xinjian/xinde/index.php?r=user/ttest
in C:\xampp\htdocs\xinjian\xinde\protected\views\user\ttest.php (10)
in C:\xampp\htdocs\xinjian\xinde\protected\controllers\UserController.php (24)
in C:\xampp\htdocs\xinjian\xinde\index.php (14)
2017/07/20 11:57:27 [error] [php] array_shift() expects parameter 1 to be array, string given (C:\xampp\htdocs\xinjian\xinde\protected\views\user\ttest.php:10)
Stack trace:
#0 C:\xampp\htdocs\xinjian\framework\web\CBaseController.php(95): UserController->renderInternal()
#1 C:\xampp\htdocs\xinjian\framework\web\CController.php(869): UserController->renderFile()
#2 C:\xampp\htdocs\xinjian\xinde\protected\controllers\UserController.php(24): UserController->renderPartial()
#3 C:\xampp\htdocs\xinjian\framework\web\actions\CInlineAction.php(49): UserController->actionTtest()
#4 C:\xampp\htdocs\xinjian\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\xampp\htdocs\xinjian\framework\web\CController.php(286): UserController->runAction()
#6 C:\xampp\htdocs\xinjian\framework\web\CController.php(265): UserController->runActionWithFilters()
#7 C:\xampp\htdocs\xinjian\framework\web\CWebApplication.php(282): UserController->run()
#8 C:\xampp\htdocs\xinjian\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 C:\xampp\htdocs\xinjian\framework\base\CApplication.php(169): CWebApplication->processRequest()
#10 C:\xampp\htdocs\xinjian\xinde\index.php(14): CWebApplication->run()
REQUEST_URI=/xinjian/xinde/index.php?r=user/ttest
in C:\xampp\htdocs\xinjian\xinde\protected\views\user\ttest.php (10)
in C:\xampp\htdocs\xinjian\xinde\protected\controllers\UserController.php (24)
in C:\xampp\htdocs\xinjian\xinde\index.php (14)
2017/07/20 11:57:54 [error] [php] array_shift() expects parameter 1 to be array, null given (C:\xampp\htdocs\xinjian\xinde\protected\views\user\ttest.php:10)
Stack trace:
#0 C:\xampp\htdocs\xinjian\framework\web\CBaseController.php(95): UserController->renderInternal()
#1 C:\xampp\htdocs\xinjian\framework\web\CController.php(869): UserController->renderFile()
#2 C:\xampp\htdocs\xinjian\xinde\protected\controllers\UserController.php(24): UserController->renderPartial()
#3 C:\xampp\htdocs\xinjian\framework\web\actions\CInlineAction.php(49): UserController->actionTtest()
#4 C:\xampp\htdocs\xinjian\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\xampp\htdocs\xinjian\framework\web\CController.php(286): UserController->runAction()
#6 C:\xampp\htdocs\xinjian\framework\web\CController.php(265): UserController->runActionWithFilters()
#7 C:\xampp\htdocs\xinjian\framework\web\CWebApplication.php(282): UserController->run()
#8 C:\xampp\htdocs\xinjian\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 C:\xampp\htdocs\xinjian\framework\base\CApplication.php(169): CWebApplication->processRequest()
#10 C:\xampp\htdocs\xinjian\xinde\index.php(14): CWebApplication->run()
REQUEST_URI=/xinjian/xinde/index.php?r=user/ttest
in C:\xampp\htdocs\xinjian\xinde\protected\views\user\ttest.php (10)
in C:\xampp\htdocs\xinjian\xinde\protected\controllers\UserController.php (24)
in C:\xampp\htdocs\xinjian\xinde\index.php (14)
2017/07/20 12:00:07 [error] [php] implode(): Invalid arguments passed (C:\xampp\htdocs\xinjian\xinde\protected\views\user\ttest.php:20)
Stack trace:
#0 C:\xampp\htdocs\xinjian\framework\web\CBaseController.php(95): UserController->renderInternal()
#1 C:\xampp\htdocs\xinjian\framework\web\CController.php(869): UserController->renderFile()
#2 C:\xampp\htdocs\xinjian\xinde\protected\controllers\UserController.php(24): UserController->renderPartial()
#3 C:\xampp\htdocs\xinjian\framework\web\actions\CInlineAction.php(49): UserController->actionTtest()
#4 C:\xampp\htdocs\xinjian\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\xampp\htdocs\xinjian\framework\web\CController.php(286): UserController->runAction()
#6 C:\xampp\htdocs\xinjian\framework\web\CController.php(265): UserController->runActionWithFilters()
#7 C:\xampp\htdocs\xinjian\framework\web\CWebApplication.php(282): UserController->run()
#8 C:\xampp\htdocs\xinjian\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 C:\xampp\htdocs\xinjian\framework\base\CApplication.php(169): CWebApplication->processRequest()
#10 C:\xampp\htdocs\xinjian\xinde\index.php(14): CWebApplication->run()
REQUEST_URI=/xinjian/xinde/index.php?r=user/ttest
in C:\xampp\htdocs\xinjian\xinde\protected\views\user\ttest.php (20)
in C:\xampp\htdocs\xinjian\xinde\protected\controllers\UserController.php (24)
in C:\xampp\htdocs\xinjian\xinde\index.php (14)
2017/07/21 04:39:07 [error] [php] Cannot use a scalar value as an array (C:\xampp\htdocs\xinjian\xinde\protected\views\user\ttest.php:6)
Stack trace:
#0 C:\xampp\htdocs\xinjian\framework\web\CController.php(869): UserController->renderFile()
#1 C:\xampp\htdocs\xinjian\xinde\protected\controllers\UserController.php(24): UserController->renderPartial()
#2 C:\xampp\htdocs\xinjian\framework\web\actions\CInlineAction.php(49): UserController->actionTtest()
#3 C:\xampp\htdocs\xinjian\framework\web\CController.php(308): CInlineAction->runWithParams()
#4 C:\xampp\htdocs\xinjian\framework\web\CController.php(286): UserController->runAction()
#5 C:\xampp\htdocs\xinjian\framework\web\CController.php(265): UserController->runActionWithFilters()
#6 C:\xampp\htdocs\xinjian\framework\web\CWebApplication.php(282): UserController->run()
#7 C:\xampp\htdocs\xinjian\framework\web\CWebApplication.php(141): CWebApplication->runController()
#8 C:\xampp\htdocs\xinjian\framework\base\CApplication.php(169): CWebApplication->processRequest()
#9 C:\xampp\htdocs\xinjian\xinde\index.php(14): CWebApplication->run()
REQUEST_URI=/xinjian/xinde/index.php?r=user/ttest
in C:\xampp\htdocs\xinjian\xinde\protected\views\user\ttest.php (6)
in C:\xampp\htdocs\xinjian\xinde\protected\controllers\UserController.php (24)
in C:\xampp\htdocs\xinjian\xinde\index.php (14)
2017/07/21 04:41:24 [error] [php] Illegal string offset 'xingming' (C:\xampp\htdocs\xinjian\xinde\protected\views\user\ttest.php:16)
Stack trace:
#0 C:\xampp\htdocs\xinjian\framework\web\CController.php(869): UserController->renderFile()
#1 C:\xampp\htdocs\xinjian\xinde\protected\controllers\UserController.php(24): UserController->renderPartial()
#2 C:\xampp\htdocs\xinjian\framework\web\actions\CInlineAction.php(49): UserController->actionTtest()
#3 C:\xampp\htdocs\xinjian\framework\web\CController.php(308): CInlineAction->runWithParams()
#4 C:\xampp\htdocs\xinjian\framework\web\CController.php(286): UserController->runAction()
#5 C:\xampp\htdocs\xinjian\framework\web\CController.php(265): UserController->runActionWithFilters()
#6 C:\xampp\htdocs\xinjian\framework\web\CWebApplication.php(282): UserController->run()
#7 C:\xampp\htdocs\xinjian\framework\web\CWebApplication.php(141): CWebApplication->runController()
#8 C:\xampp\htdocs\xinjian\framework\base\CApplication.php(169): CWebApplication->processRequest()
#9 C:\xampp\htdocs\xinjian\xinde\index.php(14): CWebApplication->run()
REQUEST_URI=/xinjian/xinde/index.php?r=user/ttest
in C:\xampp\htdocs\xinjian\xinde\protected\views\user\ttest.php (16)
in C:\xampp\htdocs\xinjian\xinde\protected\controllers\UserController.php (24)
in C:\xampp\htdocs\xinjian\xinde\index.php (14)
2017/07/21 05:18:53 [error] [php] Cannot use a scalar value as an array (C:\xampp\htdocs\xinjian\xinde\protected\views\user\ttest.php:6)
Stack trace:
#0 C:\xampp\htdocs\xinjian\framework\web\CController.php(869): UserController->renderFile()
#1 C:\xampp\htdocs\xinjian\xinde\protected\controllers\UserController.php(24): UserController->renderPartial()
#2 C:\xampp\htdocs\xinjian\framework\web\actions\CInlineAction.php(49): UserController->actionTtest()
#3 C:\xampp\htdocs\xinjian\framework\web\CController.php(308): CInlineAction->runWithParams()
#4 C:\xampp\htdocs\xinjian\framework\web\CController.php(286): UserController->runAction()
#5 C:\xampp\htdocs\xinjian\framework\web\CController.php(265): UserController->runActionWithFilters()
#6 C:\xampp\htdocs\xinjian\framework\web\CWebApplication.php(282): UserController->run()
#7 C:\xampp\htdocs\xinjian\framework\web\CWebApplication.php(141): CWebApplication->runController()
#8 C:\xampp\htdocs\xinjian\framework\base\CApplication.php(169): CWebApplication->processRequest()
#9 C:\xampp\htdocs\xinjian\xinde\index.php(14): CWebApplication->run()
REQUEST_URI=/xinjian/xinde/index.php?r=user/ttest
in C:\xampp\htdocs\xinjian\xinde\protected\views\user\ttest.php (6)
in C:\xampp\htdocs\xinjian\xinde\protected\controllers\UserController.php (24)
in C:\xampp\htdocs\xinjian\xinde\index.php (14)
2017/07/21 08:07:58 [error] [php] Cannot use a scalar value as an array (C:\xampp\htdocs\xinjian\xinde\protected\views\user\ttest.php:25)
Stack trace:
#0 C:\xampp\htdocs\xinjian\framework\web\CController.php(869): UserController->renderFile()
#1 C:\xampp\htdocs\xinjian\xinde\protected\controllers\UserController.php(24): UserController->renderPartial()
#2 C:\xampp\htdocs\xinjian\framework\web\actions\CInlineAction.php(49): UserController->actionTtest()
#3 C:\xampp\htdocs\xinjian\framework\web\CController.php(308): CInlineAction->runWithParams()
#4 C:\xampp\htdocs\xinjian\framework\web\CController.php(286): UserController->runAction()
#5 C:\xampp\htdocs\xinjian\framework\web\CController.php(265): UserController->runActionWithFilters()
#6 C:\xampp\htdocs\xinjian\framework\web\CWebApplication.php(282): UserController->run()
#7 C:\xampp\htdocs\xinjian\framework\web\CWebApplication.php(141): CWebApplication->runController()
#8 C:\xampp\htdocs\xinjian\framework\base\CApplication.php(169): CWebApplication->processRequest()
#9 C:\xampp\htdocs\xinjian\xinde\index.php(14): CWebApplication->run()
REQUEST_URI=/xinjian/xinde/index.php?r=user/ttest
in C:\xampp\htdocs\xinjian\xinde\protected\views\user\ttest.php (25)
in C:\xampp\htdocs\xinjian\xinde\protected\controllers\UserController.php (24)
in C:\xampp\htdocs\xinjian\xinde\index.php (14)
2017/07/21 08:09:34 [error] [php] Cannot use a scalar value as an array (C:\xampp\htdocs\xinjian\xinde\protected\views\user\ttest.php:25)
Stack trace:
#0 C:\xampp\htdocs\xinjian\framework\web\CController.php(869): UserController->renderFile()
#1 C:\xampp\htdocs\xinjian\xinde\protected\controllers\UserController.php(24): UserController->renderPartial()
#2 C:\xampp\htdocs\xinjian\framework\web\actions\CInlineAction.php(49): UserController->actionTtest()
#3 C:\xampp\htdocs\xinjian\framework\web\CController.php(308): CInlineAction->runWithParams()
#4 C:\xampp\htdocs\xinjian\framework\web\CController.php(286): UserController->runAction()
#5 C:\xampp\htdocs\xinjian\framework\web\CController.php(265): UserController->runActionWithFilters()
#6 C:\xampp\htdocs\xinjian\framework\web\CWebApplication.php(282): UserController->run()
#7 C:\xampp\htdocs\xinjian\framework\web\CWebApplication.php(141): CWebApplication->runController()
#8 C:\xampp\htdocs\xinjian\framework\base\CApplication.php(169): CWebApplication->processRequest()
#9 C:\xampp\htdocs\xinjian\xinde\index.php(14): CWebApplication->run()
REQUEST_URI=/xinjian/xinde/index.php?r=user/ttest
in C:\xampp\htdocs\xinjian\xinde\protected\views\user\ttest.php (25)
in C:\xampp\htdocs\xinjian\xinde\protected\controllers\UserController.php (24)
in C:\xampp\htdocs\xinjian\xinde\index.php (14)
2017/07/23 17:24:45 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'bee.chengji' doesn't exist. The SQL statement executed was: select * from chengji.
in E:\xampp\htdocs\xinde\protected\controllers\UserController.php (22)
in E:\xampp\htdocs\xinde\index.php (14)
2017/07/23 17:24:45 [error] [exception.CDbException] CDbException: CDbCommand failed to execute the SQL statement: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'bee.chengji' doesn't exist. The SQL statement executed was: select * from chengji in E:\xampp\htdocs\framework\db\CDbCommand.php:541
Stack trace:
#0 E:\xampp\htdocs\framework\db\CDbCommand.php(395): CDbCommand->queryInternal('fetchAll', Array, Array)
#1 E:\xampp\htdocs\xinde\protected\controllers\UserController.php(22): CDbCommand->queryAll(true)
#2 E:\xampp\htdocs\framework\web\actions\CInlineAction.php(49): UserController->actionTtest()
#3 E:\xampp\htdocs\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 E:\xampp\htdocs\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 E:\xampp\htdocs\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 E:\xampp\htdocs\framework\web\CWebApplication.php(282): CController->run('ttest')
#7 E:\xampp\htdocs\framework\web\CWebApplication.php(141): CWebApplication->runController('user/ttest')
#8 E:\xampp\htdocs\framework\base\CApplication.php(169): CWebApplication->processRequest()
#9 E:\xampp\htdocs\xinde\index.php(14): CApplication->run()
#10 {main}
REQUEST_URI=/xinde/index.php?r=user/ttest
---
2017/07/23 17:25:38 [error] [php] array_push() expects parameter 1 to be array, null given (E:\xampp\htdocs\xinde\protected\views\user\ttest.php:76)
Stack trace:
#0 E:\xampp\htdocs\framework\web\CBaseController.php(95): UserController->renderInternal()
#1 E:\xampp\htdocs\framework\web\CController.php(869): UserController->renderFile()
#2 E:\xampp\htdocs\xinde\protected\controllers\UserController.php(24): UserController->renderPartial()
#3 E:\xampp\htdocs\framework\web\actions\CInlineAction.php(49): UserController->actionTtest()
#4 E:\xampp\htdocs\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 E:\xampp\htdocs\framework\web\CController.php(286): UserController->runAction()
#6 E:\xampp\htdocs\framework\web\CController.php(265): UserController->runActionWithFilters()
#7 E:\xampp\htdocs\framework\web\CWebApplication.php(282): UserController->run()
#8 E:\xampp\htdocs\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 E:\xampp\htdocs\framework\base\CApplication.php(169): CWebApplication->processRequest()
#10 E:\xampp\htdocs\xinde\index.php(14): CWebApplication->run()
REQUEST_URI=/xinde/index.php?r=user/ttest
in E:\xampp\htdocs\xinde\protected\views\user\ttest.php (76)
in E:\xampp\htdocs\xinde\protected\controllers\UserController.php (24)
in E:\xampp\htdocs\xinde\index.php (14)
2017/07/23 18:31:25 [error] [php] Undefined variable: series2 (E:\xampp\htdocs\xinde\protected\views\user\ttest.php:59)
Stack trace:
#0 E:\xampp\htdocs\framework\web\CController.php(869): UserController->renderFile()
#1 E:\xampp\htdocs\xinde\protected\controllers\UserController.php(24): UserController->renderPartial()
#2 E:\xampp\htdocs\framework\web\actions\CInlineAction.php(49): UserController->actionTtest()
#3 E:\xampp\htdocs\framework\web\CController.php(308): CInlineAction->runWithParams()
#4 E:\xampp\htdocs\framework\web\CController.php(286): UserController->runAction()
#5 E:\xampp\htdocs\framework\web\CController.php(265): UserController->runActionWithFilters()
#6 E:\xampp\htdocs\framework\web\CWebApplication.php(282): UserController->run()
#7 E:\xampp\htdocs\framework\web\CWebApplication.php(141): CWebApplication->runController()
#8 E:\xampp\htdocs\framework\base\CApplication.php(169): CWebApplication->processRequest()
#9 E:\xampp\htdocs\xinde\index.php(14): CWebApplication->run()
REQUEST_URI=/xinde/index.php?r=user/ttest
in E:\xampp\htdocs\xinde\protected\views\user\ttest.php (59)
in E:\xampp\htdocs\xinde\protected\controllers\UserController.php (24)
in E:\xampp\htdocs\xinde\index.php (14)
2017/07/23 19:47:33 [error] [php] array_push() expects parameter 1 to be array, null given (E:\xampp\htdocs\xinde\protected\views\user\ttest.php:76)
Stack trace:
#0 E:\xampp\htdocs\framework\web\CBaseController.php(95): UserController->renderInternal()
#1 E:\xampp\htdocs\framework\web\CController.php(869): UserController->renderFile()
#2 E:\xampp\htdocs\xinde\protected\controllers\UserController.php(24): UserController->renderPartial()
#3 E:\xampp\htdocs\framework\web\actions\CInlineAction.php(49): UserController->actionTtest()
#4 E:\xampp\htdocs\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 E:\xampp\htdocs\framework\web\CController.php(286): UserController->runAction()
#6 E:\xampp\htdocs\framework\web\CController.php(265): UserController->runActionWithFilters()
#7 E:\xampp\htdocs\framework\web\CWebApplication.php(282): UserController->run()
#8 E:\xampp\htdocs\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 E:\xampp\htdocs\framework\base\CApplication.php(169): CWebApplication->processRequest()
#10 E:\xampp\htdocs\xinde\index.php(14): CWebApplication->run()
REQUEST_URI=/xinde/index.php?r=user/ttest
in E:\xampp\htdocs\xinde\protected\views\user\ttest.php (76)
in E:\xampp\htdocs\xinde\protected\controllers\UserController.php (24)
in E:\xampp\htdocs\xinde\index.php (14)
2017/07/23 19:48:15 [error] [php] Undefined variable: data4 (E:\xampp\htdocs\xinde\protected\views\user\ttest.php:99)
Stack trace:
#0 E:\xampp\htdocs\framework\web\CController.php(869): UserController->renderFile()
#1 E:\xampp\htdocs\xinde\protected\controllers\UserController.php(24): UserController->renderPartial()
#2 E:\xampp\htdocs\framework\web\actions\CInlineAction.php(49): UserController->actionTtest()
#3 E:\xampp\htdocs\framework\web\CController.php(308): CInlineAction->runWithParams()
#4 E:\xampp\htdocs\framework\web\CController.php(286): UserController->runAction()
#5 E:\xampp\htdocs\framework\web\CController.php(265): UserController->runActionWithFilters()
#6 E:\xampp\htdocs\framework\web\CWebApplication.php(282): UserController->run()
#7 E:\xampp\htdocs\framework\web\CWebApplication.php(141): CWebApplication->runController()
#8 E:\xampp\htdocs\framework\base\CApplication.php(169): CWebApplication->processRequest()
#9 E:\xampp\htdocs\xinde\index.php(14): CWebApplication->run()
REQUEST_URI=/xinde/index.php?r=user/ttest
in E:\xampp\htdocs\xinde\protected\views\user\ttest.php (99)
in E:\xampp\htdocs\xinde\protected\controllers\UserController.php (24)
in E:\xampp\htdocs\xinde\index.php (14)
2017/07/23 19:48:30 [error] [php] Undefined variable: xAxis1 (E:\xampp\htdocs\xinde\protected\views\user\ttest.php:126)
Stack trace:
#0 E:\xampp\htdocs\framework\web\CController.php(869): UserController->renderFile()
#1 E:\xampp\htdocs\xinde\protected\controllers\UserController.php(24): UserController->renderPartial()
#2 E:\xampp\htdocs\framework\web\actions\CInlineAction.php(49): UserController->actionTtest()
#3 E:\xampp\htdocs\framework\web\CController.php(308): CInlineAction->runWithParams()
#4 E:\xampp\htdocs\framework\web\CController.php(286): UserController->runAction()
#5 E:\xampp\htdocs\framework\web\CController.php(265): UserController->runActionWithFilters()
#6 E:\xampp\htdocs\framework\web\CWebApplication.php(282): UserController->run()
#7 E:\xampp\htdocs\framework\web\CWebApplication.php(141): CWebApplication->runController()
#8 E:\xampp\htdocs\framework\base\CApplication.php(169): CWebApplication->processRequest()
#9 E:\xampp\htdocs\xinde\index.php(14): CWebApplication->run()
REQUEST_URI=/xinde/index.php?r=user/ttest
in E:\xampp\htdocs\xinde\protected\views\user\ttest.php (126)
in E:\xampp\htdocs\xinde\protected\controllers\UserController.php (24)
in E:\xampp\htdocs\xinde\index.php (14)
2017/07/23 19:49:44 [error] [php] Undefined variable: series1 (E:\xampp\htdocs\xinde\protected\views\user\ttest.php:140)
Stack trace:
#0 E:\xampp\htdocs\framework\web\CController.php(869): UserController->renderFile()
#1 E:\xampp\htdocs\xinde\protected\controllers\UserController.php(24): UserController->renderPartial()
#2 E:\xampp\htdocs\framework\web\actions\CInlineAction.php(49): UserController->actionTtest()
#3 E:\xampp\htdocs\framework\web\CController.php(308): CInlineAction->runWithParams()
#4 E:\xampp\htdocs\framework\web\CController.php(286): UserController->runAction()
#5 E:\xampp\htdocs\framework\web\CController.php(265): UserController->runActionWithFilters()
#6 E:\xampp\htdocs\framework\web\CWebApplication.php(282): UserController->run()
#7 E:\xampp\htdocs\framework\web\CWebApplication.php(141): CWebApplication->runController()
#8 E:\xampp\htdocs\framework\base\CApplication.php(169): CWebApplication->processRequest()
#9 E:\xampp\htdocs\xinde\index.php(14): CWebApplication->run()
REQUEST_URI=/xinde/index.php?r=user/ttest
in E:\xampp\htdocs\xinde\protected\views\user\ttest.php (140)
in E:\xampp\htdocs\xinde\protected\controllers\UserController.php (24)
in E:\xampp\htdocs\xinde\index.php (14)
2017/07/23 20:03:10 [error] [php] array_push() expects parameter 1 to be array, null given (E:\xampp\htdocs\xinde\protected\views\user\ttest.php:52)
Stack trace:
#0 E:\xampp\htdocs\framework\web\CBaseController.php(95): UserController->renderInternal()
#1 E:\xampp\htdocs\framework\web\CController.php(869): UserController->renderFile()
#2 E:\xampp\htdocs\xinde\protected\controllers\UserController.php(24): UserController->renderPartial()
#3 E:\xampp\htdocs\framework\web\actions\CInlineAction.php(49): UserController->actionTtest()
#4 E:\xampp\htdocs\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 E:\xampp\htdocs\framework\web\CController.php(286): UserController->runAction()
#6 E:\xampp\htdocs\framework\web\CController.php(265): UserController->runActionWithFilters()
#7 E:\xampp\htdocs\framework\web\CWebApplication.php(282): UserController->run()
#8 E:\xampp\htdocs\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 E:\xampp\htdocs\framework\base\CApplication.php(169): CWebApplication->processRequest()
#10 E:\xampp\htdocs\xinde\index.php(14): CWebApplication->run()
REQUEST_URI=/xinde/index.php?r=user/ttest
in E:\xampp\htdocs\xinde\protected\views\user\ttest.php (52)
in E:\xampp\htdocs\xinde\protected\controllers\UserController.php (24)
in E:\xampp\htdocs\xinde\index.php (14)
2017/07/23 20:13:46 [error] [php] Undefined variable: xAxis1 (E:\xampp\htdocs\xinde\protected\views\user\ttest.php:141)
Stack trace:
#0 E:\xampp\htdocs\framework\web\CController.php(869): UserController->renderFile()
#1 E:\xampp\htdocs\xinde\protected\controllers\UserController.php(24): UserController->renderPartial()
#2 E:\xampp\htdocs\framework\web\actions\CInlineAction.php(49): UserController->actionTtest()
#3 E:\xampp\htdocs\framework\web\CController.php(308): CInlineAction->runWithParams()
#4 E:\xampp\htdocs\framework\web\CController.php(286): UserController->runAction()
#5 E:\xampp\htdocs\framework\web\CController.php(265): UserController->runActionWithFilters()
#6 E:\xampp\htdocs\framework\web\CWebApplication.php(282): UserController->run()
#7 E:\xampp\htdocs\framework\web\CWebApplication.php(141): CWebApplication->runController()
#8 E:\xampp\htdocs\framework\base\CApplication.php(169): CWebApplication->processRequest()
#9 E:\xampp\htdocs\xinde\index.php(14): CWebApplication->run()
REQUEST_URI=/xinde/index.php?r=user/ttest
in E:\xampp\htdocs\xinde\protected\views\user\ttest.php (141)
in E:\xampp\htdocs\xinde\protected\controllers\UserController.php (24)
in E:\xampp\htdocs\xinde\index.php (14)
2017/07/23 20:39:37 [error] [php] Undefined offset: 5 (E:\xampp\htdocs\xinde\protected\views\user\ttest.php:64)
Stack trace:
#0 E:\xampp\htdocs\framework\web\CController.php(869): UserController->renderFile()
#1 E:\xampp\htdocs\xinde\protected\controllers\UserController.php(24): UserController->renderPartial()
#2 E:\xampp\htdocs\framework\web\actions\CInlineAction.php(49): UserController->actionTtest()
#3 E:\xampp\htdocs\framework\web\CController.php(308): CInlineAction->runWithParams()
#4 E:\xampp\htdocs\framework\web\CController.php(286): UserController->runAction()
#5 E:\xampp\htdocs\framework\web\CController.php(265): UserController->runActionWithFilters()
#6 E:\xampp\htdocs\framework\web\CWebApplication.php(282): UserController->run()
#7 E:\xampp\htdocs\framework\web\CWebApplication.php(141): CWebApplication->runController()
#8 E:\xampp\htdocs\framework\base\CApplication.php(169): CWebApplication->processRequest()
#9 E:\xampp\htdocs\xinde\index.php(14): CWebApplication->run()
REQUEST_URI=/xinde/index.php?r=user/ttest
in E:\xampp\htdocs\xinde\protected\views\user\ttest.php (64)
in E:\xampp\htdocs\xinde\protected\controllers\UserController.php (24)
in E:\xampp\htdocs\xinde\index.php (14)
2017/07/23 20:40:31 [error] [php] Undefined offset: 30 (E:\xampp\htdocs\xinde\protected\views\user\ttest.php:65)
Stack trace:
#0 E:\xampp\htdocs\framework\web\CController.php(869): UserController->renderFile()
#1 E:\xampp\htdocs\xinde\protected\controllers\UserController.php(24): UserController->renderPartial()
#2 E:\xampp\htdocs\framework\web\actions\CInlineAction.php(49): UserController->actionTtest()
#3 E:\xampp\htdocs\framework\web\CController.php(308): CInlineAction->runWithParams()
#4 E:\xampp\htdocs\framework\web\CController.php(286): UserController->runAction()
#5 E:\xampp\htdocs\framework\web\CController.php(265): UserController->runActionWithFilters()
#6 E:\xampp\htdocs\framework\web\CWebApplication.php(282): UserController->run()
#7 E:\xampp\htdocs\framework\web\CWebApplication.php(141): CWebApplication->runController()
#8 E:\xampp\htdocs\framework\base\CApplication.php(169): CWebApplication->processRequest()
#9 E:\xampp\htdocs\xinde\index.php(14): CWebApplication->run()
REQUEST_URI=/xinde/index.php?r=user/ttest
in E:\xampp\htdocs\xinde\protected\views\user\ttest.php (65)
in E:\xampp\htdocs\xinde\protected\controllers\UserController.php (24)
in E:\xampp\htdocs\xinde\index.php (14)
2017/07/23 20:41:37 [error] [php] Undefined offset: 5 (E:\xampp\htdocs\xinde\protected\views\user\ttest.php:64)
Stack trace:
#0 E:\xampp\htdocs\framework\web\CController.php(869): UserController->renderFile()
#1 E:\xampp\htdocs\xinde\protected\controllers\UserController.php(24): UserController->renderPartial()
#2 E:\xampp\htdocs\framework\web\actions\CInlineAction.php(49): UserController->actionTtest()
#3 E:\xampp\htdocs\framework\web\CController.php(308): CInlineAction->runWithParams()
#4 E:\xampp\htdocs\framework\web\CController.php(286): UserController->runAction()
#5 E:\xampp\htdocs\framework\web\CController.php(265): UserController->runActionWithFilters()
#6 E:\xampp\htdocs\framework\web\CWebApplication.php(282): UserController->run()
#7 E:\xampp\htdocs\framework\web\CWebApplication.php(141): CWebApplication->runController()
#8 E:\xampp\htdocs\framework\base\CApplication.php(169): CWebApplication->processRequest()
#9 E:\xampp\htdocs\xinde\index.php(14): CWebApplication->run()
REQUEST_URI=/xinde/index.php?r=user/ttest
in E:\xampp\htdocs\xinde\protected\views\user\ttest.php (64)
in E:\xampp\htdocs\xinde\protected\controllers\UserController.php (24)
in E:\xampp\htdocs\xinde\index.php (14)
